.changeset/good-ads-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+Adds wrangler support for Vectorize, Cloudflare's new vector database, under `wrangler vectorize`.
```

Comment:
Ideally this would have more details (example commands etc...), but maybe a link to the docs would work?

Reply:
Good call. Fixing.